StaticThreadController.h

```diff
@@ -0,0 +1,65 @@
+/*
+	StaticThreadController.h - Controlls a list of Threads with different timings
+
+	Basicaly, what it does is to keep track of current Threads and run when
+	necessary.
+
+	StaticThreadController is an extended class of Thread, because of that,
+	it allows you to add a StaticThreadController inside another kind of ThreadController...
+
+	It works exact as ThreadController except you can't add or remove treads dynamically.
+
+	Created by Alex Eremin, September, 2016.
+	Released into the public domain.
+*/
+
+#ifndef StaticThreadController_h
+#define StaticThreadController_h
+
+#include "Thread.h"
+
+template <int N>
+class StaticThreadController: public Thread{
+protected:
+	Thread thread[N];
```

Comment:
It's not cool to save copies of Threads. This might go wrong:

```
Thread threadA = Thread(callback, 100);

StaticThreadController<1> stc = StaticThreadController(threadA);

// And then, to disable:
threadA.enabled = false;
```

In this case, wouldnt the Thread not be stopped, because it was "copied" into StaticThreadController?